GuildedRoseLLC/ItemsViewController.swift

```diff
@@ -10,14 +10,19 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
     override public func viewDidLoad() {
         super.viewDidLoad()
-        let items = Item.getItems()
-        configureDataSource(items: items)
-        toggleListDisplay(items: items)
+        configureDataSource()
         
         greeting.accessibilityIdentifier = "Greeting"
         itemCollectionView.accessibilityIdentifier = "itemCollectionView"
     }
     
+    override public func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let items = Item.getItems()
+        configureDataSource(items: items)
```

Comment:
I feel like there is still a disconnect on the purpose of these changes. The idea was to configure the data source once in `viewDidLoad`, and show the items in `viewWillAppear`.

To me, it feel like this line may not be needed.